### ISSUE TITLE
Fix binding of volumes

### ISSUE THREAD

**Issue by bbarker** [CornellCAC/reactive-docker#4]
Currently getting requests like this after a bit of fiddling:

```
dockerRequest: http://localhost:4243/containers/create?name=dockeropenmpi_head3
```

``` json
 {
  "Image" : "dockeropenmpi_mpi_node",
  "Cmd" : [ "/usr/sbin/sshd", "-D" ],
  "Volumes" : {
    "/data" : {
      "path" : "/data",
      "hostPath" : "/tmp"
    }
  },
  "NetworkDisabled" : false
}
```

According to the [API](https://docs.docker.com/engine/reference/api/docker_remote_api_v1.24/), this is fairly off: Volumes should only contain the container mount point and HostConfig should contain the actual bindings.

**Comment by bbarker** [comment]
It would seem that "volumes" entry is not rquired, but hostconfig is:

``` json
{
    "Image" : "dockeropenmpi_mpi_head",
    "Cmd" : [ "/usr/sbin/sshd", "-D" ],
    "Volumes": {
        "/data": {}
    },
    "HostConfig": {
        "Binds": ["/tmp:/data"]
    },
    "NetworkDisabled" : false
}
```

However, I'll plan to leave it in anyway as that seems to match the specification.

**Comment by bbarker** [comment]
Fixed in d565038b7440e22bc381899184897640137b1207.
